diff --git a/CMakeLists.txt b/CMakeLists.txt
index 18207e7..f2eca28 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,7 +30,7 @@ add_dependencies(
     )
 
 set(COMMON_LIBS
-    ${CMAKE_CURRENT_BINARY_DIR}/pcm-src/libPCM.a
+    ${CMAKE_CURRENT_BINARY_DIR}/pcm-build/src/libpcm.a
     m
     rt
     dl
diff --git a/CMakeLists.txt.in b/CMakeLists.txt.in
index bf3b0d9..bad2288 100644
--- a/CMakeLists.txt.in
+++ b/CMakeLists.txt.in
@@ -14,20 +14,19 @@ ExternalProject_Add(gnuplot-iostream
   TEST_COMMAND      ""
 )
 
-include_directories(${CMAKE_CURRENT_BINARY_DIR}/gnuplot-iostream-src)
+include_directories(${CMAKE_CURRENT_BINARY_DIR}/pcm-src/src)
 
 
 ExternalProject_Add(pcm
-  GIT_REPOSITORY    https://github.com/opcm/pcm.git
-  GIT_TAG           202107
+  GIT_REPOSITORY    https://github.com/intel/pcm.git
+  GIT_TAG           202604
   SOURCE_DIR        "${CMAKE_CURRENT_BINARY_DIR}/pcm-src"
-  BINARY_DIR        "${CMAKE_CURRENT_BINARY_DIR}/pcm-src"
-  CONFIGURE_COMMAND ""
-  UPDATE_COMMAND    git checkout Makefile
-  PATCH_COMMAND     git apply ${CMAKE_CURRENT_LIST_DIR}/dont_use_perf_in_pcm.patch
-  BUILD_COMMAND     make lib
+  BINARY_DIR        "${CMAKE_CURRENT_BINARY_DIR}/pcm-build"
+  CONFIGURE_COMMAND cmake -S ${CMAKE_CURRENT_BINARY_DIR}/pcm-src -B ${CMAKE_CURRENT_BINARY_DIR}/pcm-build
+  BUILD_COMMAND     cmake --build ${CMAKE_CURRENT_BINARY_DIR}/pcm-build --target PCM_STATIC
   INSTALL_COMMAND   ""
   TEST_COMMAND      ""
 )
 
+
 include_directories(${CMAKE_CURRENT_BINARY_DIR}/pcm-src)
 
